src/DetectChanges/BCBreak/FunctionBased/MultipleChecksOnAFunction.php

```diff
@@ -18,12 +18,12 @@ public function __construct(FunctionBased ...$checks)
         $this->checks = $checks;
     }
 
-    public function __invoke(ReflectionFunctionAbstract $fromFunction, ReflectionFunctionAbstract $toClass) : Changes
+    public function __invoke(ReflectionFunctionAbstract $fromFunction, ReflectionFunctionAbstract $toFunction) : Changes
     {
         return array_reduce(
             $this->checks,
-            function (Changes $changes, FunctionBased $check) use ($fromFunction, $toClass) : Changes {
-                return $changes->mergeWith($check->__invoke($fromFunction, $toClass));
+            function (Changes $changes, FunctionBased $check) use ($fromFunction, $toFunction) : Changes {
+                return $changes->mergeWith($check->__invoke($fromFunction, $toFunction));
```

Comment:
Is there a way to detect anonymous functions BTW?

Reply:
See description:
> Note: the issue specifies filtering out anonymous functions too, but we don't actually reflect on functions at all at the moment; it is all just `ClassBased`, `InterfaceBased` and `TraitBased` roots; plus the top level is the `CompareClasses` implementation...